sync/peerset/peer_set.go

```diff
@@ -31,23 +32,11 @@ func NewPeerSet(sessionTimeout time.Duration) *PeerSet {
 	}
 }
 
-// GetPeer finds a peer by id and returns a copy of the peer object.
-func (ps *PeerSet) GetPeer(pid peer.ID) Peer {
-	ps.lk.RLock()
-	defer ps.lk.RUnlock()
-
-	p := ps.getPeer(pid)
-	if p != nil {
-		return *p
-	}
-
-	return Peer{}
-}
-
 func (ps *PeerSet) OpenSession(pid peer.ID) *Session {
 	ps.lk.Lock()
 	defer ps.lk.Unlock()
 
+	//TODO: check if pid exists
```

Comment:
Remove this comment or update OpenSession calls to make sure it is not nil

Reply:
Better to remove the comment